### PR TITLE
Added dutch translations.

### PR DESCRIPTION
**Pull request by iamreinder**
Hi there,

I came across this library and it was exactly what I needed for a task manager kind of tool. I needed Dutch translations, though. So I added rules for `nl`. Some parts might look a lot different from the English rules. That's because it's quite common to combine the day of the week with an indication of the part of the day in Dutch. For instance woensdagavond combines woensdag (wednesday) and avond (evening).

All the tests pass. Let me know if you have any questions.

My kind regards,
Reinder.

### CODE REVIEW

File: when.go [hunk 1]
```diff
@@ -1,14 +1,15 @@
 package when
 
 import (
+	"github.com/olebedev/when/rules/nl"
```
Comment (olebedev) [inline when.go:4]: Can we move this to the section where all the other languages are being imported?

**Review by olebedev — DISMISSED** [review]
Hey @iamreinder, this looks excellent; thank you for your contribution.

Would you consider becoming the code owner for the `rules/nl` folder? Since I am not fluent in Dutch, it would be beneficial to have an expert like you overseeing that part.

File: when.go (hunk 1 above)
Comment (iamreinder) [reply when.go]: Sure! Done :-)

**Comment by iamreinder** [comment]
@olebedev I made the requested change. In regards to being code owner, I think it makes sense. What do you expect of me as a code owner for the Dutch language rules?

**Comment by olebedev** [comment]
> What do you expect of me as a code owner for the Dutch language rules?

@ iamreinder, not much, just to keep an eye on the incoming changes and verify if they still make sense for the Dutch language context. Does this sound good to you?

**Comment by iamreinder** [comment]
Yeah, sure :-)

**Comment by olebedev** [comment]
Great! Can you then please add yourself into the `CODEOWNERS` file? The change would be similar to [this](https://github.com/olebedev/when/blob/master/CODEOWNERS#L2) line but for the Dutch language.

**Comment by olebedev** [comment]
Friendly ping @iamreinder

**Review by olebedev — APPROVED** [review]
Approve it to get it in. I will add you @iamreinder into the contributors by myself later on.

Cheers,
Oleg

**Pull request merged**